tasks/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
-# tasks file for .
\ No newline at end of file
+ - import_tasks: install.yml
+   become: yes
```

Comment:
Missing trailing EOL. Also just place the contents of install.yml here

Reply:
fixed and no i will not do that since i need a configure.yml soon